src/thirdparties/css/style.css

```diff
@@ -525,7 +528,7 @@ footer p a,footer p a:hover,footer p a:focus {
     h1{
         font-size: 30px;
     }
-    .subscription-form .input-box {
```

Comment:
is this selector correct? download-form doesn't seem to be in subpscription-form